domain/src/test/java/org/oppia/android/domain/classify/InteractionObjectTestBuilder.kt

```diff
@@ -76,6 +78,33 @@ object InteractionObjectTestBuilder {
     ).build()
   }
 
+  fun createFractionForNumberWithUnits(
```

Comment:
I think we can use the already existing `createFraction` function for creating the fraction for number with units.

Reply:
I used  createFraction  but it shows error for the createNumberWithUnits as the createFraction defined before has fun createFraction() : InteractionObject and createNumberWithUnits needs to take fraction so I created another one with createFractionForNumberWithUnits() : Fraction. I couldn't think of any other solution.

Reply:
Aah I see. Actually you can get the fraction out of the interaction object. You can do something like `createFraction(values).fraction` to get the fraction out of the interaction object. Yes, it gives a null-safe return value, but I think you can handle that.

Reply:
It worked. I will remove it from the file quickly. Thanku @Sarthak2601

Reply:
Done